tox.ini

```diff
@@ -1,16 +1,47 @@
 [tox]
-envlist = py33,py34,py35,py36,coverage,flake8,isort
+envlist = {py33,py34,py35,py36,coverage}-{cext,nocext},flake8,isort
 
 [testenv]
+setenv =
+    cext: WEBSOCKETS_CEXT_BUILD=1
+    nocext: WEBSOCKETS_CEXT_BUILD=0
 deps =
     py33: asyncio
-commands = python -m unittest
+commands =
+    python setup.py clean --all
+    python -c "import glob, os; [os.remove(f) for files in [glob.glob('./websockets/' + ext) for ext in ['*.so', '*.pyd']] for f in files]"
```

Comment:
Good point about Windows support. I still think I'll stick with `rm` until there's a need for running tox on Windows.